### PR TITLE
[SHIP-5023] API Cargowise reference Endpoint description

### PR DESCRIPTION
**Pull request by SowmyaLakkegowda29**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -488,6 +488,347 @@ curl -X POST
...
+
+This endpoint can be used to send Organization/Container Number/Product Code reference data.
+How to send each of these format has been explained in this document below.
+
+Organization data:
+This is a <UniversalInterchange> request.
+XML tag <OrgHeader></OrgHeader> wraps up all the organization related details such as Organization code, name, address etc..
+
+>Example xml format when sending organization data:
+```
```
Comment (lphkaldn) [inline source/index.html.md:514]: Do we get syntax highlighting with ``` xml?

File: source/index.html.md [hunk 2]
```diff
@@ -488,6 +488,347 @@ curl -X POST
...
+
+Container reference data:
+This is a <UniversalInterchange> request.
+XML tag <UniversalShipment></UniversalShipment> wraps up all the container reference data,
+You can specify multiple containers by repeating the <Container></Container> XML tag.
+Similarly, multiple shipments can be specified by repeating <SubShipment></SubShipment> XML tag.
+
+>Example xml format when sending Container reference data:
+```
```
Comment (lphkaldn) [inline source/index.html.md:629]: see above

File: source/index.html.md [hunk 3]
```diff
@@ -488,6 +488,347 @@ curl -X POST
...
+  </Body>
+</UniversalInterchange>
+```
+
+Product code data:
+This is a <XmlInterchange> request.
+XML tag <Products></Products> wraps up all the product code related data.
+
+>Example xml format when sending product code data:
+```
```
Comment (lphkaldn) [inline source/index.html.md:701]: see above

File: source/index.html.md [hunk 4]
```diff
@@ -488,6 +488,347 @@ curl -X POST
...
+    </Products>
+  </Payload>
+</XmlInterchange>
+```
+
+Cragowise Reference endpoint can also accept SOAP message which is a Cargowise default i.e, Request that starts with tag <s: Envelope>, Or 
+you can also take the message encoded within the SOAP message and post it as a request to the Cargowise reference endpoint.
+
+>Example xml format when sending SOAP message:
+```
```
Comment (lphkaldn) [inline source/index.html.md:790]: see above

File: source/index.html.md [hunk 5]
```diff
@@ -488,6 +488,347 @@ curl -X POST
...
+                </Message>
+      </Payload>
+    </SendStreamRequest>
+  </s:Body>
+</s:Envelope>
+```
+
+>The POST /send endpoint responds with data as 'text/xml' like this
+```
```
Comment (lphkaldn) [inline source/index.html.md:818]: see above

File: source/index.html.md (hunk 1 above)
Comment (SowmyaLakkegowda29) [reply source/index.html.md:514]: Ah good spot!

**Review by lphkaldn — CHANGES_REQUESTED** [review]
I'm not sure why, but this still does not highlight the xml. Please check the characters ```xml and the do a visual check locally if the elementes are coloured.

**Review by lphkaldn — APPROVED** [review]

**Pull request merged**